### DISCUSSION TITLE
🐞 stdout logs from services (`dagger call ... up`) are truncated and have no way to retrieve

### DISCUSSION THREAD

**Discussion by nipuna-perera** [dagger/dagger#12331] [Help]
### What is the issue?

I am running a service `dagger call ... up`. The service, which is a Java app, continuously outputs to stdout. However, from the Dagger TUI, I only see a continuous tail of the log. I have no way to get to the part of the log that has "scrolled off" the screen.

This also affects the output of any "ENTRYPOINT" script. I have a substantial script that outputs a lot to stdout but I can't get to that log when I run the service or after I terminate it.

### Dagger version

dagger v0.10.3

### Steps to reproduce

Run any service with `dagger call ... up` and observe the log output the stdout.

**Comment by levlaz** [comment]
Hey @nipuna-perera! 

I think this is the case for any command by default in the TUI now.  if you preface the command with `dagger --debug call ... up` are you able to see the logs become scrollable?

**Comment by nipuna-perera** [comment]
Hey @levlaz that is what I expected. But for services, --debug didn't make it scrollable either.

**Comment by levlaz** [comment]
Thanks Nipuna, that make sense, feels like Services are a bit special in this regard.

Can you try one more thing and run `call —progress=plain` and tell me what you think of the behavior? I just tried it with my MariaDB module and its a bit closer to what I expect.

dagger call --progress=plain -m mariadb serve up

Lev Lazinskiy
Head of Solutions @ dagger.io ( http://dagger.io )
e: ***@***.***
p: 415.470.2142

**Comment by nipuna-perera** [comment]
Just getting back to this now. on v0.11.0 I am trying to run with `--progress=plain` but my pipeline is frozen after "Connected to engine..."

According to @vito this may not work at all with 0.11.0
https://discord.com/channels/707636530424053791/1120503349599543376/1227001321324810471

**Comment by jedevc** [comment]
See also https://github.com/dagger/dagger/issues/6553 and https://github.com/dagger/dagger/issues/7377.

**Comment by nvanheuverzwijn** [comment]
Sorry for reviving this, but using progress=plain works. It's a bit ugly because the whole output is directly in the terminal but you get the logs. Would be nicer if we could scroll in the TUI.
